### PR TITLE
CN-4834, CN-4900: Security overview screen redesign

### PR DESCRIPTION
**Pull request by CrnogoracIvan**
## Checklist (Mandatory):

- [x] I have named the PR correctly (Check PR1)
- [x] Check the commit’s or even all commits message styles matches our requested structure. (issue number, etc.) (Check CR10)
- [x] I have tested new code locally (Check CR3)
- [x] I have done a smoke test of the larger feature locally (Check CR3)
- [x] I have added at least two reviewers (Check PR5)
- [x] I have assigned the PR to myself (Check PR4)
- [x] I have merged develop (parent) branch into my feature branch and resolved all conflicts (Check PR7)

## Checklist (Optional):

- [ ] I have added necessary documentation, including instructions how to test (Check CR8)
- [ ] I have added tests that prove my fix/implementation is effective or that my feature works
- [ ] I have written the code according to the adopted style guide? (Check CR7)

## Additional notes

> What does this implement/fix? Explain your changes. (optional)
> Are there any scripts that need running (eg. yarn install)?
> Are there any edge cases that need checking?
> Are there any specific test instructions?

[MVPWorkshop Pull Request & Code Review Guidelines](https://docs.google.com/document/d/1TcuU1JBhlGZLtpy0osSNinXQ2piP_yrDuRcXdqaOS_U/edit#heading=h.5b6kpdr4btq8)

### CODE REVIEW

File: app/components/atoms/Atoms.stories.js [hunk 1]
```diff
@@ -66,5 +68,8 @@ storiesOf("Atoms", module)
   .add("OfflineMode", OfflineModeStories)
   .add("PaymentListItem", PaymentListItemStories)
   .add("ProgressBar", ProgressBarStories)
+  .add("SecurityStrengthMeter", SecurityStrengthMeter)
+  .add("SecurityScoreGauge", SecurityScoreGauge)
+  .add("ProgressBar", ProgressBarStories)
```
Comment (fjovakaric) [inline app/components/atoms/Atoms.stories.js]: duplicate

File: app/components/molecules/ToggleInfoCard/ToggleInfoCard.js [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+import Icon from "../../atoms/Icon/Icon";
+import CelSwitch from "../../atoms/CelSwitch/CelSwitch";
+// import StoryWrapper from "../../atoms/StoryWrapper/StoryWrapper";
+
+@connect(
+  state => ({
+    interestRates: state.generalData.interestRates,
+  }),
+  dispatch => ({ actions: bindActionCreators(appActions, dispatch) })
+)
```
Comment (fjovakaric) [inline app/components/molecules/ToggleInfoCard/ToggleInfoCard.js]: remove redux

File: app/components/molecules/ToggleInfoCard/ToggleInfoCard.js [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+      activeType: "disabled",
+    };
+  }
+
+  componentDidMount() {
+    const { status } = this.props;
+    if (status === "enabled") {
+      this.setState({ activeType: "enabled" });
+    }
+  }
```
Comment (fjovakaric) [inline app/components/molecules/ToggleInfoCard/ToggleInfoCard.js]: this can be set in the constructor

File: app/components/molecules/ToggleInfoCard/ToggleInfoCard.js [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+@connect(
+  state => ({
+    interestRates: state.generalData.interestRates,
+  }),
+  dispatch => ({ actions: bindActionCreators(appActions, dispatch) })
+)
+class ToggleInfoCard extends Component {
+  static propTypes = {
+    mode: PropTypes.oneOf(["info", "toggle"]).isRequired,
+    status: PropTypes.oneOf(["info", "enabled", "disabled"]),
```
Comment (fjovakaric) [inline app/components/molecules/ToggleInfoCard/ToggleInfoCard.js]: remove the info case, and create separate atom/molecule

File: app/components/molecules/ToggleInfoCard/ToggleInfoCard.js [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+import STYLES from "../../../constants/STYLES";
+import CelText from "../../atoms/CelText/CelText";
+import { getTheme } from "../../../utils/styles-util";
+import ToggleInfoCardStyle from "./ToggleInfoCard.styles";
+import Icon from "../../atoms/Icon/Icon";
+import CelSwitch from "../../atoms/CelSwitch/CelSwitch";
+
+class ToggleInfoCard extends Component {
+  static propTypes = {
+    status: PropTypes.oneOf(["enabled"]),
```
Comment (fjovakaric) [inline app/components/molecules/ToggleInfoCard/ToggleInfoCard.js]: enabled (true/false)

File: app/components/molecules/ToggleInfoCard/ToggleInfoCard.js [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+  };
+  static defaultProps = {};
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      loading: false,
+      switchActivated: false,
+      activeType: "disabled",
```
Comment (fjovakaric) [inline app/components/molecules/ToggleInfoCard/ToggleInfoCard.js]: enabled: props.enabled

File: app/components/molecules/ToggleInfoCard/ToggleInfoCard.js (hunk 3 above)
Comment (fjovakaric) [reply app/components/molecules/ToggleInfoCard/ToggleInfoCard.js]: delete?

File: app/components/molecules/ToggleInfoCard/ToggleInfoCard.js [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+  };
+
+  render() {
+    const style = ToggleInfoCardStyle();
+
+    const { subtitle } = this.props;
+    const { activeType } = this.state;
+    const cardParams = this.getCardProps();
+    const backgroundColorStyle =
+      activeType === "enabled" ? { backgroundColor: STYLES.COLORS.GREEN } : {};
```
Comment (fjovakaric) [inline app/components/molecules/ToggleInfoCard/ToggleInfoCard.js]: check this?
